tests/test_http_graphql_query.py

```diff
@@ -2794,6 +2796,19 @@ def test_graphql_functional_variables_39(self):
                 variables={'limit': '1'},
             )
 
+    def test_graphql_functional_variables_40(self):
+        with self.assertRaisesRegex(
+                edgedb.QueryError,
+                r"Only scalar defaults are allowed\. "
+                r"Variable 'val' has non-scalar default value\."):
+            self.graphql_query(r"""
+                query($val: FilterFloat = {eq: 3.0}) {
+                    User(filter: {score: $val}) {
+                        id,
+                    }
+                }
+            """)
+
```

Comment:
Should we add a couple of functional end to end tests here to test the new functioality?  @vpetrovykh

Reply:
We have tests of the rewrite in the rust. And current functional tests have both constants in the graphql and errors on that constants. So I feel that tests are fine as they are.